README.md

```diff
@@ -14,9 +14,19 @@ Online documentation is available at https://mozilla.github.io/mozanalysis/
 pip install mozanalysis
 ```
 
+## Local Installation
+```bash
+# Create and activate a python virtual environment.
+python3 -m venv venv/
+source venv/bin/activate
+pip install -r requirements.txt
```

Comment:
We need a `pip install -e .` line following this. The instructions as written only install the dependencies, not the actual package. And yeah... Jetstream's instructions are wrong too.

Reply:
Lol that's what I get for copying and pasting

Reply:
added to the README